docs/src/componentDocs/ToolbarMenu/playground/ToolbarMenuConfig.tsx

```diff
@@ -0,0 +1,27 @@
+import { ComponentType } from '../../../__types__';
+
+export const toolbarMenuConfig: ComponentType = {
+    componentName: 'Toolbar Menu',
+    props: [
+        {
+            propName: 'icon',
+            inputType: 'select',
+            inputValue: '<GradeA />',
+            options: ['<GradeA />', '<GradeB />'],
```

Comment:
I will nominate these options to be [`<Place />`](https://fonts.google.com/icons?selected=Material%20Icons%3Aplace%3A), [`<Home />`](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled&icon.query=home) and `undefined` instead